bigfile_editor/known_types.py

```diff
@@ -23,7 +23,25 @@
             "name" : "Character definitions",
             "0" : "Axel4 Character",
             "1" : "Blaze4 Character",
-            "2" : "Cherry Character"
+            "2" : "Cherry Character",
+            "3": "Floyd Character",
+            "4": "Adam4 Character",
+            "5": "Blaze1 Character",
+            "6": "Adam1 Character",
+            "7": "Axel1 Character",
+            "8": "Skate2 Character",
+            "9": "Blaze2 Character",
+            "10": "Axel2 Character",
+            "11": "Max2 Character",
+            "12": "Axel3 Character",
+            "13": "Blaze3 Character",
+            "14": "Roo Character",
+            "15": "Skate3 Character",
+            "16": "Zan Character",
+            "17": "Shiva3 Character",
+            "18": "Estel Character",
+            "19": "Max4 Character",
+            "20": "Shiva4 Character",
```

Comment:
Small syntax error here; the final element should not have a comma after it.

Reply:
@JadingTsunami Np, comma removed